Frontend/ui-driver/src/Components/IndividualRideCard/View.purs

```diff
@@ -221,49 +319,45 @@ sourceAndDestination =
                 , height $ V 16
                 , width $ V 14
                 ]
-              , textView
+              , textView $
                 [ PrestoList.textHolder "destination"
-                , textSize FontSize.a_14
                 , layoutGravity "center_vertical"
                 , padding (Padding 10 0 70 2)
-                , fontStyle $ FontStyle.regular LanguageStyle
                 , maxLines 1
                 , ellipsize true
-                , color Color.black700
-                ]
+                , color Color.black900
+                ] <> FontStyle.paragraphText TypoGraphy
               ]
         ]
       
     ]
 
-rideWithDetails :: forall w. PrestoDOM (Effect Unit) w 
-rideWithDetails = 
-  textView
-    [ PrestoList.textHolder "rideDistance"
-    , PrestoList.visibilityHolder "ride_distance_visibility"
-    , textSize FontSize.a_14
+distanceAndCustomerName :: forall w. PrestoDOM (Effect Unit) w 
+distanceAndCustomerName = 
+  linearLayout
```

Comment:
it can just return text view why this linearLayout is required give margin to textView directly